systemtest/pom.xml

```diff
@@ -122,19 +122,26 @@
         <dependency>
             <groupId>io.vertx</groupId>
             <artifactId>vertx-web-client</artifactId>
-            <version>${vertx.version}</version>
+            <version>3.7.0</version>
```

Comment:
do you need the specific 3.7.0 version for any reason? Vert.x Kafka client or something like that? Btw we are on 3.7.1 now.

Reply:
Changed to global version used in the project